lib/membrane_matroska/muxer.ex

```diff
@@ -0,0 +1,396 @@
+defmodule Membrane.Matroska.Muxer do
+  @moduledoc """
+  Filter element for muxing Matroska files.
+
+  It accepts an arbitrary number of Opus, VP8, VP9 or H264 streams and outputs a bytestream in Matroska format containing those streams.
+
+  Muxer guidelines
+  https://www.matroskaproject.org/docs/container/
+  """
+  use Bitwise
+  use Bunch
+
+  use Membrane.Filter
+
+  alias Membrane.{Buffer, RemoteStream}
+  alias Membrane.{Opus, VP8, VP9, MP4}
+  alias Membrane.Matroska.Parser.Codecs
+  alias Membrane.Matroska.Serializer
+  alias Membrane.Matroska.Serializer.Helper
+  alias Membrane.MP4.Payload.AVC1
+
+  def_options title: [
+                spec: String.t(),
+                default: "Membrane Matroska file",
+                description: "Title to be used in the `Segment/Info/Title` element"
+              ],
+              date: [
+                spec: nil | integer(),
```

Comment:
My intention was this, so that you don't need to bother ourself with converting this thing to gregorian milliseconds or whatever is needed. Especially that what it is isn't indicated in the description
```suggestion
                spec: nil | DateTime.t(),
```